app/src/components/Admin/UserInfo/UserGenericContainerComponent.tsx

```diff
@@ -0,0 +1,136 @@
+import GenericModalComponent from '../../GenericModalComponent';
+/*
+This code bassicly extends the GenericModalCompoent
+Where we re use the GenericMopdalComponent, but with the help of the, we add
+some additional fields to the modal, like the name and email of the user.
+
+!!!! Please DONT try to merge the UserGenericContainerComponent with the GenericModalComponent!!!!!
+
+*/
+interface UserGenericContainerComponentProps {
+  isOpen: boolean;
+  onClose: () => void;
+  title: string;
+  contentText: string;
+  confirmBtnText: string;
+  onConfirm: () => void;
+  userId: string;
+  token: string;
+  onHandleStatus: () => void;
+  userDetails: any;
+  isReject: boolean;
+  Loading: boolean;
+}
+
+const UserGenericContainerComponent: React.FC<UserGenericContainerComponentProps> = ({
+  isOpen,
+  onClose,
+  title,
+  contentText,
+  confirmBtnText,
+  userId,
+  token,
+  onHandleStatus,
+  userDetails,
+  isReject,
+  
+}) => {
+  const [reason, setReason] = useState('');
+  const [isInputValid, setIsInputValid] = useState(true);
+  const [Loading, setLoading] = useState(false);
+
+  if (!isOpen) return null;
+
+  const handleAction = async () => {
+    if (isReject && !reason.trim()) {
+      setIsInputValid(false);
+      return;
+    }
+
+    setLoading(true);
+
+    try {
+      if (isReject) {
+        await AuthServices.RejectApplication(userId, reason);
+        toast.success("Application rejected!");
+      } else {
+        await AuthServices.AcceptApplication(userId);
+        await AdminServices.changeUserRole(userId, token, 'creator');
+        toast.success("Application approved!");
+      }
+      onClose();
+      onHandleStatus();
+    } catch (error) {
+      console.error("Failed to process application:", error);
+    } finally {
+      setLoading(false);
+    }
+  };
+
+  return (
+    <GenericModalComponent
+      title={title}
+      contentText={contentText}
+      confirmBtnText={confirmBtnText}
+      cancelBtnText="Voltar"
+      onClose={onClose}
+      isVisible={isOpen}
+      onConfirm={handleAction}
+      isConfirmDisabled={isReject && !isInputValid || Loading}
+      Loading={Loading}
+      className={isReject ? "adaptive-modal" : ""}
+      width={"800px"} // Add this line to define the width
+    >
+      <div className="grid grid-cols-1 md:grid-cols-2 gap-4">
+        <div className="flex flex-col bg-white p-4 md:p-6 rounded-l-lg mt-4 relative">
+          <dt className="text-[#166276] text-base font-bold font-['Lato']">Nome</dt>
+          <dd id="name" className="text-base font-['Montserrat'] text-gray-900 break-all">
+            {userDetails.firstName} {userDetails.lastName}
+          </dd>
+          <style>{`
+            .relative::after {
+              content: '';
+              position: absolute;
+              right: 0;
+              top: 20%;
+              height: 60%;
+              width: 1px;
+              background-color: #E7F3F6;
+            }
+          `}</style>
```

Comment:
I dont like the style tag, im pretty sure it adds this styling globally and changes the tailwind class for the entire project.
So this needs to be changed in some way, preferably with tailwind classes 😅 

I think you might be able to put another div right after the relative one with the absolute class on it.